config/katello.migrations/20210304102446_add_proxy_shellhooks_plugin.rb

```diff
@@ -0,0 +1 @@
+answers['foreman_proxy::plugin::shellhooks'] ||= false
```

Comment:
I think you also want to add a migration for foreman-proxy-content.migrations and add it to the answers file for that scenario as well.

Reply:
Good point, added